app/src/components/Tree.js

```diff
@@ -71,15 +72,33 @@ function Tree({ links, nodes }) {
   });
 
   return (
-    <div className="svg-container">
+    <div className="svg-container" style={{ overflow: "visible" }}>
       <svg
         version="1.1"
         viewBox="0 0 600 600"
         preserveAspectRatio="xMinYMin meet"
         className="svg-content"
         ref={ref}
         // style={{width: "100%", height: "100%"}}
+        style={{ overflow: "visible" }}
       >
+        <image
+          style={{
+            backgroundImage: `url(${image.hdurl})`,
+            // backgroundColor: "pink",
+            backgroundSize: "cover",
+            opacity: ".32",
+            overflow: "visible",
+            enableBackground: "new",
+            transform: `translateY(calc(-66vh + 66vmin)) translateX(calc(-55vw + 55vmin))`,
+            position: "absolute",
+            top: 0,
+          }}
+          // width={window.innerWidth}
+          // width="100%"
+          // height={window.innerHeight}
+          href={image.hdurl}
```

Comment:
I'm a little confused. You seem to be calling your NASA image in multiple places. It's just supposed to be in the background of your app, right?

Reply:
I tried both `backgroundImage` as well as a straight `image` element: only the latter was functional. Background images did not work for the svg or its container because of the hacky thing I did to make the svg responsive. In the comments were attempts to make things work that did not. The easiest solution would be to just target the `html` element with css, but the image url can only be loaded via javascript. I might try and see if it would work on `index.html` later on ( when the new image of the day breaks the layout )

Reply:
A separate image element ( or pseudo element ) allows the image to have lowered opacity without affecting the other svg elements.

Reply:
Hmmm... do you know you can target the html element from JavaScript as well, right? And then apply the image url you loaded? And we can make it responsive without having a separate svg element. And there's css and/or inline styling you can add to lower the opacity of your image without it affecting other elements. Let me know if this is something you'd like to do and we can pair on it. :]

Reply:
Yes. There are a lot of solutions to make it better, I’ve been advised to create all the front-to-back connections before focusing too much on front-end detail. The PR is just making the image visible (the commented background image styles are actually places where it did not work!) and looking for ideas on hiding keys.

For ideas, I have thought of creating a pseudo-element for the html element, but you cannot target pseudo elements in javascript. It’s possible to `position: fixed` or `absolute` an image element as well. What I have ideally wanted was for the image to scale 1:1 with the svg but it’s too much of a time sink. I’m just waiting for a new image to load every day and adjust the inline css if need be.